code/src/Submit.js

```diff
@@ -0,0 +1,10 @@
+/*import React, {useState} from 'react';
+
+const Submit = () => {
+    return (
+
+    
+    )
+}
+
+export default Submit; */
```

Comment:
Delete component if you are not using it 😊

Reply:
oops, forgot that! Thanks! 😄